deployments/helm-chart/templates/controller-service.yaml

```diff
@@ -29,6 +32,9 @@ spec:
   {{- end }}
 {{- end }}
   type: {{ .Values.controller.service.type }}
+  {{- if .Values.controller.service.ipFamilyPolicy }}
+  ipFamilyPolicy: {{ .Values.controller.service.ipFamilyPolicy }}
+  {{- end }}
```

Comment:
I think it would make sense to add `ipFamilies` too, what do you think?

Reply:
I've added it below.